harness/determined/cli/job.py

```diff
@@ -100,6 +87,89 @@ def update(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def process_updates(args: Namespace) -> None:
+    session = setup_session(args)
+    for arg in args.operation:
+        inputs = validate_operation_args(arg)
+        _single_update(session=session, **inputs)
+
+
+def _single_update(
+    job_id: str,
+    session: Session,
+    priority: str = "",
+    weight: str = "",
+    resource_pool: str = "",
+    behind_of: str = "",
+    ahead_of: str = "",
+) -> None:
+    update = bindings.v1QueueControl(
+        jobId=job_id,
+        priority=int(priority) if priority != "" else None,
```

Comment:
hmm perhaps just convert `""` -> `None` for all `operand[1]` in input validation instead of repeating it for all 5 arguments here?

Reply:
yeah I did a bunch of things to try and satisfy the checker. Another problem that comes up is that ``priority`` and ``weight`` needed to be converted to integers , but the linter complains if you try to assign ``None`` to a declared ``int`` type. So the most elegant way to do this for ``priority`` and ``weight`` is to clean up the args here. At that point, I thought maybe I should just keep it all consistent.